src/Symfony/Bridge/Monolog/Tests/Processor/ConsoleCommandProcessorTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Monolog\Tests\Processor;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\Monolog\Processor\ConsoleCommandProcessor;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Event\ConsoleEvent;
+use Symfony\Component\Console\Input\InputInterface;
+
+class ConsoleCommandProcessorTest extends TestCase
+{
+    const TEST_ARGUMENTS = array('test' => 'argument');
```

Comment:
Could you turn the visibility of theses const to `private` ?

Reply:
OK, changed.